leap-consent-ui/src/main/java/gov/hhs/onc/leap/backend/fhir/client/utils/FHIRProvenance.java

```diff
@@ -0,0 +1,65 @@
+package gov.hhs.onc.leap.backend.fhir.client.utils;
+
+import gov.hhs.onc.leap.backend.fhir.client.HapiFhirServer;
+import org.hl7.fhir.r4.model.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Service
+public class FHIRProvenance {
+    private String PROVENANCE_AGENT = "LEAP Consent Management Service";
+    private String PROVENANCE_AGENT_SYSTEM = "http://terminology.hl7.org/CodeSystem/provenance-participant-type";
+    private String PROVENANCE_AGENT_CODE = "assembler";
+    private String PROVENANCE_AGENT_DISPLAY = "Assember";
+
+    private String PROVENANCE_ACTIVITY_SYSTEM = "http://terminology.hl7.org/CodeSystem/v3-DataOperation";
+    private String PROVENANCE_ACTIVITY_CODE= "CREATE";
+    private String PROVENANCE_ACTIVITY_DISPLAY = "create";
```

Comment:
These must be final and can be also static (if we change the visibility)

Reply:
thanks done